internal/saucecloud/cucumber.go

```diff
@@ -87,7 +87,7 @@ func (r *CucumberRunner) getSuiteNames() string {
 	return strings.Join(names, ", ")
 }
 
-func (r *CucumberRunner) runSuites(fileURIs []string) bool {
+func (r *CucumberRunner) runSuites(fileURIs map[uploadType]string) bool {
```

Comment:
Would it be easier if the signature changed to `apps []string, otherApps []string`?

Reply:
IMO `app` can be `app.zip` from batch frameworks side or `App` from Espresso/XCUITest. `otherApps` can be `config.zip` and `node_modules`. I'll update like this strategy.